go/digest/BUILD.bazel

```diff
@@ -16,7 +16,6 @@ go_test(
     srcs = ["digest_test.go"],
     embed = [":go_default_library"],
     deps = [
-        "@com_github_bazelbuild_remote_apis//build/bazel/remote/execution/v2:go_default_library",
```

Comment:
Why did this go away? Doesn't it still depend on RE (repb)?

Reply:
Not directly, no.